pyfar/dsp/filter.py

```diff
@@ -679,3 +680,274 @@ def _shelve(signal, frequency, gain, order, shelve_type, sampling_rate, kind):
         # return the filtered signal
         signal_filt = filt.process(signal)
         return signal_filt
+
+
+def center_frequencies_fractional_octaves(
+        num_fractions=1, frequency_range=(20, 20e3), return_cutoff=False):
+    """Return the octave center frequencies according to the IEC 61260:1:2014
+    standard. For numbers of fractions other than 1 and 3, only the exact
+    center frequencies are returned, since nominal frequencies are not
+    specified by corresponding standards.
+
+    Parameters
+    ----------
+    num_fractions : int, 1
+        The number of bands a octaves is divided into.
```

Comment:
Is having the default value listed as the type common? I think numpy recommends to mention it at the end of the definition, e.g.:
```suggestion
    num_fractions : int, optional
        The number of bands octaves are divided into. E.g., 1 refers to octave bands and 3 to third octave bands. The default is 1.
```